test/UnitTest/data/cosmoscollections/dataelements/d03b4a04-f0df-4ead-be92-aa7a68959dab.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "d03b4a04-f0df-4ead-be92-aa7a68959dab",
+  "instanceGuid": "31d0941f-6d56-40a6-b4a4-b7fe18ccff30",
+  "dataType": "default",
+  "contentType": "text/plain; charset=utf-8",
+  "blobStoragePath": "tdd/endring-av-navn/649388f0-a2c0-4774-bd11-c870223ed819/data/d03b4a04-f0df-4ead-be92-aa7a68959dab",
```

Comment:
Invalid instance guid in blob storage path

Reply:
Fixed